cmake/fmtlib.cmake

```
@@ -0,0 +1,20 @@
+include(FetchContent)
+
+FetchContent_Declare(
+	fmtlib
+	PREFIX "${CMAKE_BINARY_DIR}/deps"
+	DOWNLOAD_DIR "${CMAKE_SOURCE_DIR}/deps/downloads"
+	DOWNLOAD_NAME fmt-7.1.3.tar.gz
```

It's already at 8.0.1 now. I think we should take the latest version.
```suggestion
	DOWNLOAD_NAME fmt-8.0.1.tar.gz
```